### DISCUSSION TITLE
1.1 Wholesale Exercise Question

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#22] [Q&A] [answered]
I know this question is more me not knowing the basic equation and not so much not knowing how to code correctly, but still would like some guidance on this example. Thanks for the help!

![image](https://user-images.githubusercontent.com/106542680/171316813-1ef279b1-6b17-4ab2-bf99-810918a0bebc.png)

**Comment by feliciones** [comment, marked as answer]
Hello,

Your code is all correct except for one small error. What you did is found what 40% is of the price of all the books, you need to find the price with 40% **off**. You need to do n * price * (1 - discount), or 1 - 0.4 which is 0.6. I hope this helps.

Best,
Simon

**Reply by ecieszynski** [reply]
Thank you for the correction. Definitely just a silly mistake because I was working pretty late at night. Thanks again
